### ISSUE TITLE
Multiple File Upload Failing

### ISSUE THREAD

**Issue by sammhill** [projectsend/projectsend#512]
We have an issue where on our installation (IIS 10, Windows Server 2016, PHP 5.6.31), that when we upload multiple files and assign them to users, when we click continue after assigning them, the server returns a blank page, and if we go to Manage Files, usually 3 or 4 out of 9 files have been successfully uploaded and assigned.

The odd thing, is that when we re-attempt the upload with the files that failed, they work absolutely fine.

I have tried the following:

- Changed our Maximum POST size to 4096M
- Changed our Upload Maximum file size to 3072
- Changed our Maximum files to upload to 100
- Changed our Maximum execution time to 600
- Changed our Memory Limit to 1024M

Does anyone have any ideas on why this may be happening? All of the files have similar naming, and they're all the same file type ranging from 6KB to ~250MB

If any logs will come in useful, please let me know!

Cheers!

**Comment by LaurenAdelle** [comment]
I am running into the same problem.  Someone tries to upload 15+ files at a time (which used to work fine) and now only certain ones come through.  But at first we get a blank screen after assigning the files.  Then I go into file management and some of the files are there but only a couple 2-3.

- We have tried deleting all the old files in the person's portal

- We tried making a new web portal for the person and it is still acting the same.

Any advice on how to get the files Uploading again?

LaurenAdelle
